frontend/src/App.scss

```diff
@@ -36,9 +36,22 @@ a:visited {
   background-color: $ttahub-blue;
 }
 
+smart-hub-title {
```

Comment:
missing the dot, so I don't think this is applying to anything

maybe we can lose the entire rule if that's the case

Reply:
Nope - need the dot. What is the process for fixing it?

Reply:
the 1rem declaration was the mobile size, it is larger at larger widths.

Reply:
Yep. When I went down to mobile size, it looks like the default font size for views smaller than 64rem is 1rem.